lib/functions

```diff
@@ -210,3 +210,22 @@ function configure_service {
     chmod +x /etc/init.d/$service_script
     update-rc.d $service_script defaults
 }
+
+# Install OpenDayLight SDN controller - Download and install ODL
+function install_ODL {
+    if [ ! -d /opt/opendaylight ]; then
+        mkdir /opt/opendaylight/
+        wget https://nexus.opendaylight.org/content/repositories/public/org/opendaylight/integration/distribution-karaf/0.5.3-Boron-SR3/distribution-karaf-0.5.3-Boron-SR3.tar.gz -P /opt/
```

Comment:
I think that the value "0.5.3-Boron-SR3" should be stored in a variable for future changes.